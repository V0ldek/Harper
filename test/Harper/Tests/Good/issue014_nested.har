main :: Integer;
main = {
    var (n :: Integer) = 0;
    var (result :: Integer) = 0;

    {
        (n :: Integer) = 1;
        {
            (n :: Integer) = 2;
            {
                var (n :: Integer) = 3;
                {
                    (n :: Integer) = 4;
                    result += n;
                }
                result += n;
            }
            result += n;
        }
        result += n;
    }
    result += n;

    return result;
};