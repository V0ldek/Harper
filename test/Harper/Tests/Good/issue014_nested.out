Execution ended with value:
10